fdbserver/Resolver.actor.cpp

```diff
@@ -244,6 +244,18 @@ ACTOR Future<Void> versionReady(Resolver* self, ProxyRequestsInfo* proxyInfo, Ve
 	}
 }
 
+// Check if the given set of tags contain any tags other than the log router tags.
+// Used to check if a given "ResolveTransactionBatchRequest" corresponds to an
+// empty commit message or not.
+static bool hasNonLogRouterTags(std::set<Tag>& allTags) {
```

Comment:
you can make `allTags` const to signal intent

Reply:
Makes sense. Done now.